app/containers/connectedAuthorShow/index.tsx

```diff
@@ -222,7 +223,8 @@ class ConnectedAuthorShow extends React.PureComponent<ConnectedAuthorShowProps,
             currentAffiliation: author.lastKnownAffiliation || "",
             bio: author.bio || "",
             website: author.webPage || "",
-            email: author.email,
+            email: author.email || "",
+            isEmailHidden: author.isEmailHidden,
```

Comment:
I think we need a default value for `isEmailHidden` because it can be `undefined`.